pkg/cephfs/controllerserver.go

```diff
@@ -99,7 +90,7 @@ func (cs *controllerServer) CreateVolume(ctx context.Context, req *csi.CreateVol
 	return &csi.CreateVolumeResponse{
 		Volume: &csi.Volume{
 			Id:            string(volId),
-			CapacityBytes: sz,
+			CapacityBytes: req.GetCapacityRange().GetRequiredBytes(),
```

Comment:
github doesn't allow me to comment unchanged the code :)

according to csi spec comment [here](https://github.com/container-storage-interface/spec/blob/master/csi.proto#L402), at least one of required bytes or limits bytes, so it is entirely possible the required bytes are 0 while limited bytes still indicate a quota. But since [kubernetes csi provision](https://github.com/kubernetes-csi/external-provisioner/blob/master/pkg/controller/controller.go#L445) only uses required bytes, let's take this a solution.

Reply:
Correct, my bad! I'll take this into consideration in future PRs.